### PR TITLE
Actually fixes wormhole jaunters, and no it was not my fault.

### PR DESCRIPTION
**Pull request by silicons**

### CODE REVIEW

File: code/modules/mining/equipment/wormhole_jaunter.dm [hunk 1]
```diff
@@ -89,20 +89,18 @@
...
-		if(do_teleport(M, hard_target, 6))
-			// KERPLUNK
-			playsound(M,'sound/weapons/resonator_blast.ogg',50,1)
-			if(iscarbon(M))
-				var/mob/living/carbon/L = M
-				L.Knockdown(60)
-				if(ishuman(L))
-					shake_camera(L, 20, 1)
-					addtimer(CALLBACK(L, /mob/living/carbon.proc/vomit), 20)
+	if(M.anchored && !istype(M, /obj/mecha) && !ismob(M))
```
Comment (ChangelingRain) [inline code/modules/mining/equipment/wormhole_jaunter.dm]: no check for mech_sized var?

Comment (kevinz000) [inline code/modules/mining/equipment/wormhole_jaunter.dm]: it's set to on anyways so why bother :P

Comment (kevinz000) [inline code/modules/mining/equipment/wormhole_jaunter.dm]: wait nvm im dumb il fix

File: code/modules/mining/equipment/wormhole_jaunter.dm [hunk 2]
```diff
@@ -89,20 +89,18 @@
 	mech_sized = TRUE //save your ripley
 
 /obj/effect/portal/wormhole/jaunt_tunnel/teleport(atom/movable/M)
-	if(istype(M, /obj/effect))
+	if(istype(M, /obj/effect) || !istype(M))
```
Comment (Cyberboss) [inline code/modules/mining/equipment/wormhole_jaunter.dm]: In what situation would this istype fail?

**Comment by Cyberboss** [comment]
@kevinz000

File: code/modules/mining/equipment/wormhole_jaunter.dm (hunk 2 above)
Comment (kevinz000) [inline code/modules/mining/equipment/wormhole_jaunter.dm]: uh
uh
uh
good point!

**Pull request merged**